### PR TITLE
Added java-simple and java-spring-boot to gsc test examples.

### PR DESCRIPTION
**Pull request by sreeharikax**

### CODE REVIEW

File: ci/lib/stage-test-gsc.jenkinsfile [hunk 1]
```diff
@@ -40,6 +40,22 @@ stage('Test') {
...
         '''
     } catch (Exception e) {}
+    try {
+        sh '''
+            docker run --device=/dev/sgx_enclave \
+                -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+                gsc-openjdk-11-java-simple 2>&1 | tee openjdk-simple_result
+        '''
+    } catch (Exception e) {}
+    try {
```
Comment (anjalirai-intel) [inline ci/lib/stage-test-gsc.jenkinsfile:50]: Add single line space wherever applicable

**Review by anjalirai-intel — CHANGES_REQUESTED** [review]

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 2]
```diff
@@ -90,4 +90,38 @@ stage('Build Workloads') {
             ./gsc info-image gsc-build-gramine-bash-test
         '''
     } catch (Exception e) {}
+    
+    // Build java-simple Workload
+    try {
+        sh '''
+            if [ "${distro_ver}" = "debian:11" ]
```
[the remark this answers is not in the document]
Comment (sreeharikax) [reply ci/lib/stage-build-gsc.jenkinsfile]: Modified.

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 3]
```diff
@@ -90,4 +90,38 @@ stage('Build Workloads') {
...
+                cd Examples/java-simple
+                docker build -t openjdk-11-java-simple .
+                cd ../../
+                ./gsc build openjdk-11-java-simple Examples/java-simple/java-simple.manifest
+                ./gsc sign-image openjdk-11-java-simple enclave-key.pem 
+                ./gsc info-image gsc-openjdk-11-java-simple
+            fi
+        '''
+    } catch (Exception e) {}
+    // Build java-spring-boot Workload
```
[the remark this answers is not in the document]
Comment (sreeharikax) [reply ci/lib/stage-build-gsc.jenkinsfile]: Added.

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 4]
```diff
@@ -90,4 +90,38 @@ stage('Build Workloads') {
...
+    try {
+        sh '''
+            if [ "${distro_ver}" = "debian:11" ]
+            then
+                sudo apt-get install -y openjdk-11-jdk
```
[the remark this answers is not in the document]
Comment (sreeharikax) [reply ci/lib/stage-build-gsc.jenkinsfile]: Done

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 5]
```diff
@@ -90,4 +90,38 @@ stage('Build Workloads') {
...
+                wget https://services.gradle.org/distributions/gradle-7.6.4-bin.zip
```
[the remark this answers is not in the document]
Comment (sreeharikax) [reply ci/lib/stage-build-gsc.jenkinsfile]: Done

File: ci/lib/stage-test-gsc.jenkinsfile [hunk 6]
```diff
@@ -40,6 +40,22 @@ stage('Test') {
                 gsc-build-gramine-bash-test ${runtime_args} 2>&1 | tee build_gramine_bash_result
         '''
     } catch (Exception e) {}
+    try {
```
[the remark this answers is not in the document]
Comment (sreeharikax) [reply ci/lib/stage-test-gsc.jenkinsfile:44]: Added

File: ci/lib/stage-test-gsc.jenkinsfile (hunk 1 above)
Comment (sreeharikax) [reply ci/lib/stage-test-gsc.jenkinsfile:52]: Added

**Pull request merged**